### ISSUE TITLE
False positive in tabs: Line indented incorrectly; expected at least 2 tabs, found 2

### ISSUE THREAD

**Issue by davidperezgar** [WordPress/WordPress-Coding-Standards#1990]
## Bug Description
It's getting false positives in line indents.

## Minimal Code Snippet
```
		echo do_shortcode( '[my_subcatboxes]' );
		echo do_shortcode( '[my_sticky_posts]' );
```

For bugs with fixers: How was the code fixed? How did you expect the code to be fixed?

## Error Code
Line indented incorrectly; expected at least 2 tabs, found 2 phpcs

## Environment

| Question               | Answer
| ------------------------| -------
| PHP version             | 7.3.24
| PHP_CodeSniffer version | 3.3.1
| WPCS version            | 2.3.0
| WPCS install type       | Composer global
| IDE (if relevant)       | VSCode

**Comment by jrfnl** [comment]
@davidperezgar Thanks for reporting this, but I'm not sure what you expect.

Two things here:
1. The issue cannot be reproduced with the code sample you provided. Not with the exact same PHP/PHPCS/WPCS combination as you've listed above, nor with the current develop version of PHPCS/WPCS.
2. And even if it could be reproduced, this is not an issue which can be solved in the WordPressCS repo as the sniff involved is a sniff from PHP_CodeSniffer itself.
    As per the [CONTRIBUTING guidelines](https://github.com/WordPress/WordPress-Coding-Standards/blob/develop/.github/CONTRIBUTING.md#upstream-issues), issues with PHPCS native sniffs should be reported in that repo.
    You can see whether a sniff is from WPCS itself or from elsewhere by running PHPCS with the `-s` flag to see the error code.

Having said that, I'd strongly recommend you upgrade your PHPCS version before reporting anything upstream as you are using an outdated version. The current version is `3.6.0` and your issue may well be solved already.

**Comment by jrfnl** [comment]
Closing for lack of response.
If this is still an issue, please provide the additional information requested and we can reopen the issue.
